images/base/Dockerfile

```diff
@@ -80,10 +80,19 @@ RUN clean-install \
 COPY 10-limits.conf 10-restart.conf /etc/systemd/system/containerd.service.d/
 
 # debug containerd version and create default config
-# additionally, disable some plugins we don't use / support
+# additionally:
+# - disable some plugins we don't use / support
+# - setup the "test-handler" used by kubernetes runtime class testing
 RUN containerd --version \
     && mkdir -p /etc/containerd \
-    && echo 'disabled_plugins = ["aufs", "btrfs", "zfs"]' > /etc/containerd/config.toml \
+    && ( \
+        echo "# disable plugins we don't / can't support" \
```

Comment:
bah, this doesn't work right in a dockerfile

Reply:
https://github.com/kubernetes-sigs/kind/pull/895

Reply:
:-)